recipes/deepac/meta.yaml

```diff
@@ -44,7 +48,7 @@ test:
     - deepac.explain
     - deepac.gwpa
   commands:
-    - deepac --help
+    - deepac --version | grep -qF '{{ version }}'
```

Comment:
Let's fetch the artifacts after the build so we can test the container from the CI.